docs/source/_toctree.yml

```
@@ -1,8 +1,30 @@
-- sections:
-  - local: index
-    title: Bits & Bytes
+- title: Get started
+  sections:
+  - local: introduction
+    title: Introduction
```

```suggestion
  - local: index
    title: index
```
No need for introduction, this should be converted to "index", see my comment below